bokehjs/src/lib/models/math_text/math_text.ts

```diff
@@ -57,7 +58,27 @@ export class CDNProvider extends MathJaxProvider  {
   }
 }
 
-// TODO: module/bundle provider
+export class BundleProvider extends MathJaxProvider  {
+  _mathjax: typeof MathJax | null
+
+  get MathJax(): typeof MathJax | null {
+    return this._mathjax
+  }
+
+  async fetch(): Promise<void> {
+    this.status = "loading"
+
+    try {
+      const mathjax = await load_module(import("./mathjax"))
+      this._mathjax = mathjax
+
+      this.status = "loaded"
+      this.ready.emit()
+    } catch (error) {
+      this.status = "failed"
+    }
+  }
+}
 
 const default_provider: MathJaxProvider = new CDNProvider()
```

Comment:
So, we finally have a bundle, but what now? How do we use it? What's the point of having a CDN provider and include a bundle at the same time?

Reply:
Nice question, this slip through. I'm thinking of solving it by using a config from Python and having the BundleProvider as default, but is there a need to use the CDN provider ever again?